### ISSUE TITLE
Additional Head to the architecture 

### ISSUE THREAD

**Issue by abhigoku10** [argusswift/YOLOv4-pytorch#233]
@argusswift  thanks for sharing the code base i had the following queries 
1. can we add an additional head to the existing code base which performs multi-task classification like gender,age classification for each person box detected?
2. Can we add an additional head to the whole pipeline which performs scene classification / multiattribute classification  

Please share your thoughts on thsi

**Comment by argusswift** [comment]
> @argusswift thanks for sharing the code base i had the following queries
> 
> 1. can we add an additional head to the existing code base which performs multi-task classification like gender,age classification for each person box detected?
> 2. Can we add an additional head to the whole pipeline which performs scene classification / multiattribute classification
> 
> Please share your thoughts on thsi

Of course, as long as you associate the label (like gender and age you mentioned above) with the corresponding picture. The main points of modification are: 1. The last layer of convolution of the head network needs more channels to adapt to this change. 2. The change of this attribute needs to be taken into account in label encoding and decoding processing.
